src/app/app.ts

```diff
@@ -47,7 +47,7 @@ export class App {
 
   showGamePage() {
     if (!this.gamePage) {
-      this.gamePage = new GamePage(this, this.game)
+      this.gamePage = new GamePage(this, this.game, this.audio)
```

Comment:
Hmm im not sure but 'this' contains game and audio so those two additional args are not necessary